components/layout/mediaQuery/README.md

```diff
@@ -0,0 +1,67 @@
+# LayoutMediaQuery
+
+`LayoutMediaQuery` is a wrapper that will pass through a _function as a child_ an object with the breakpoints matched according to the width of the parent container
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-layout-media-query --save
+```
+
+## Usage
+
+### Basic usage (default breakpoints)
+
+```javascript
+import LayoutMediaQuery from '@s-ui/react-layout-media-query'
+
+<LayoutMediaQuery>
+  {({M, L, XL}) => {
+    if (XL) return <p>LargeDesktop</p>
```

Comment:
Just wondering how this approach might be implemented in our current layouts..
Do you mean that we should return one component or another depending the matching breakpoint??
If so.. this is not the way responsive is.. if you replace components

Reply:
Hi @davecarter, this is one of our biggest concerns. We want to avoid having two apps, but we are reaching our limits because of designs that we need to be compliant with, like the MA home page or some IJ headers. Mostly this component will be used to set one or another prop, component replacement should be the last approach we think of.